Monika After Story/game/script-greetings.rpy

```diff
@@ -2897,6 +2914,8 @@ label greeting_returned_home_morethan5mins_normalplus_dlg:
     m 1eub "Even if I couldn't really see anything, knowing that I was really right there with you..."
     m 2eua "Well, it felt really great!"
     m 5eub "Let's do this again soon, okay?"
+    if not mas_isplayer_bday() and persistent._mas_player_bday_decor:
```

Comment:
```suggestion
    if persistent._mas_player_bday_decor and not mas_isplayer_bday():
```
very minor, but having this in reverse order takes advantage of short circuiting. Not player bday will be true 364/365 days of the year, while bday_decor will only be True on 1 day. 

Having not bday as the first condition means that for almost the entire year, we have to check that conditoin and the decor condition, so 2 conditions. 

Having the decor check first willl only check both conditions if decor is True. If decor is False, python wont bother checking the 2nd var since this is an And comparison.